nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -412,3 +425,38 @@ def get_statistic_points(self, model: TModel) -> StatisticPointsContainer:
                                                       tensor_collector=stat_collector,
                                                       algorithm=MinMaxQuantization))
         return output
+
+    def _apply_model_type_pass(self, model_type: Optional[ModelType], quantizer_setup: SingleConfigQuantizerSetup,
+                               nncf_graph: NNCFGraph, device: TargetDevice) -> None:
+        """
+        Applies changes in-place into quantizer setup based on model_type and device parameters.
+
+        :param model_type: Model type parameter.
+        :param quantizer_setup: Quantizer setup which considered to update.
+        :param nncf_graph: Instance of NNCFGraph.
+        :param device: Target device.
+        :return: None
+        """
+        if model_type == ModelType.TRANSFORMER and device in [TargetDevice.ANY, TargetDevice.CPU, TargetDevice.GPU]:
+            to_remove_quantizers = set()
```

Comment:
Why do we need to remove quantizers from the `quantizer_setup`? I've assumed that we've added the layers to the ignored scope in the previous step.

Reply:
We need to filter the quantizers on MatMuls after Softmax

Reply:
Ok, filter, not remove. The question is still valid.